include/ola/io/Serial.h

```diff
@@ -70,19 +95,24 @@ bool UIntToSpeedT(uint32_t value, speed_t *output);
  * (the default) or UUCP locking.  See: ./configure --enable-uucp-locking.
  *
  * This fails-fast, it we can't get the lock immediately, we'll return false.
+ *
+ * @see ReleaseSerialPortLock()
  */
-bool AcquireLockAndOpen(const std::string &path, int oflag, int *fd);
+bool AcquireLockAndOpenSerialPort(const std::string &path, int oflag, int *fd);
 
 /**
- * @brief Remove a UUCP lock file for the device.
+ * @brief Release a lock for the serial port
  * @param path The path to unlock.
  *
- * The lock is only removed if the PID matches.
+ * If UUCP locking was used (see AcquireLockAndOpenSerialPort), the lockfile
```

Comment:
I can't remember if this will auto-link, or whether you need to give the full namespace path for that.

Reply:
I've checked, and added parentheses but not the whole path.  This makes it link correctly.